Source: Alpine Linux (adapted)
Upstream: No
Reason: Fixes compilation on i686

--- a/Make.defaults
+++ b/Make.defaults
@@ -156,11 +156,11 @@ ifneq (mingw32,$(findstring mingw32, $(GCCMACHINE)))
 endif
 
 ifeq (FreeBSD, $(findstring FreeBSD, $(OS)))
-CFLAGS  += $(ARCH3264) -g -O2 -Wall -Wextra -Werror \
+CFLAGS  += $(ARCH3264) -g -O2 -Wall -Wextra \
            -fshort-wchar -fno-strict-aliasing \
            -ffreestanding -fno-stack-protector
 else
-CFLAGS  += $(ARCH3264) -g -O2 -Wall -Wextra -Werror \
+CFLAGS  += $(ARCH3264) -g -O2 -Wall -Wextra \
            -fshort-wchar -fno-strict-aliasing \
            -fno-merge-all-constants -ffreestanding -fno-stack-protector \
            -fno-stack-check
